Makefile

```diff
@@ -112,6 +112,10 @@ terraform/install: | $(BIN_DIR) guard/program/jq
 	$(@D) --version
 	@ echo "[$@]: Completed successfully!"
 
+terragrunt/install: TERRAGRUNT_VERSION ?= latest
+terragrunt/install: | $(BIN_DIR) guard/program/jq
+	@ $(MAKE) install/gh-release/$(@D) FILENAME="$(BIN_DIR)/$(@D)" OWNER=gruntwork-io REPO=$(@D) VERSION=$(TERRAGRUNT_VERSION) QUERY='.name | endswith("$(OS)_$(ARCH)")'
+
```

Comment:
I've seen a customized Makefile for the task I'm working on and it includes these additional lines for this target:

```
        chmod +x $(BIN_DIR)/$(@D)
        $(@D) --version
        @ echo "[$@]: Completed successfully!"
```

Reply:
This is also done already, in the target `install/gh-release/%`...